pkg/loop/standalone_provider_test.go

```diff
@@ -7,13 +7,13 @@ import (
 	"google.golang.org/grpc"
 
 	"github.com/smartcontractkit/chainlink-common/pkg/loop"
-	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/test"
+	pluginprovider_test "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/test/ocr2/plugin_provider"
```

Comment:
Why the alias?

Reply:
i find it easier to read. if we did eventually move to individual packages with sub packages for `/test` wouldn't it read like this?

Reply:
> if we did eventually move to individual packages with sub packages for /test wouldn't it read like this?

Yes but it would be aligned.


Another common pattern requiring no aliases is:
```suggestion
	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/ocr2/pluginprovider/pluginprovidertest"
```

Reply:
i've moved the code to pkgXXX/test where ever possible, instead of test/XXX eg median/test, ocr3/test, ...

i filed this as follow up work https://smartcontract-it.atlassian.net/browse/BCF-3039 and put a reference in the README to clean it up further.

the fact that is was possible for me to move the code around so quickly is strong evidence that this change is both good progress and sufficient for the current purposes